test/unit/spec/v2/ion/uiSchemaTransformer_spec.js

```diff
@@ -10,6 +10,178 @@ import XHRAuthenticatorEnrollOptions  from '../../../../../playground/mocks/idp/
 
 
 describe('v2/ion/uiSchemaTransformer', function () {
+  it('converts identify remidiation response', () => {
+    const rawIdentifyResponse = identifyResponse;
+    const transformedResponse  = {
```

Comment:
check my PR https://github.com/okta/okta-signin-widget/pull/1202, I did refactoring the way at constructing `transformedResponse` and assertion.